ci/prow/make_config.go

```diff
@@ -568,6 +575,9 @@ func generatePeriodic(title string, repoName string, periodicConfig yaml.MapSlic
 			if len(data.Base.Args) == 0 {
 				data.Base.Args = allPresubmitTests
 			}
+			addLabelToJob(&data.Base, "prow.k8s.io/pubsub-project", "knative-tests")
+			addLabelToJob(&data.Base, "prow.k8s.io/pubsub-topic", "knative-monitoring")
+			addLabelToJob(&data.Base, "prow.k8s.io/pubsub-runID", fmt.Sprintf("ci-%s-%s", data.Base.RepoNameForJob, jobNameSuffix))
```

Comment:
You are trying to make this string the same as the job here, right? Can you use just take the job name, instead? What if the job name changed to other format than fmt.Sprintf("ci-%s-%s", data.Base.RepoNameForJob, jobNameSuffix)

Reply:
The job name is not populated until much later after it determines the job type. I added the labels here with duplicated job name logic because I don't want to change the code flow for the initial testing as we may expand the scope to label all periodic jobs. 

I can move the name logic if we are not going to label all periodic jobs in the future. Currently, the periodic job types are `continuous`, `nightly`, `branch-ci`, `dot-release`, `auto-release`, `performance`, `performance-mesh`, `latency`, `custom-job`, `cron`, `release`, `webhook-apicoverage`. 

Do you know if there's a job type from the above list that should not be labelled?

Reply:
why don't we just use func getTestGroupName(repoName string, jobName string). In this way if there is any change in the future we don't need to deal with bugs derived from discrepancy.

We actually need the jobname to be returned from pubsub, because we need to store that in the db. If we don't return it through runID here then we need to change crier's implementation to export it.

Sorry, I don't understand your 2nd and 3rd paragraph. You can talk with me offline

Reply:
Changed the labelled job to go-coverage which is not part of this method. Let's decide on which jobs to label later.